qiita_pet/templates/study_description.html

```diff
@@ -148,11 +148,10 @@
         break;
     }
   }
-
   if (sff.length === 0 && barcodes.length === 0 && forward.length === 0 && fasta.length === 0) {
     bootstrapAlert("You need to select at least: one barcode and one forward file, one sff file or one fasta file.");
     return [false];
-  } else if (sff.length === 0 && barcodes.length != forward.length) {
+  } else if (sff.length === 0 && barcodes.length != forward.length && filetype !== 5 && filetype != "per_sample_FASTQ") {
```

Comment:
Is there any specific test that needs to be done in a `per_sample_FASTQ`? i.e. the same number of forward and reverse, in case that the user chose any reverse read?